react/features/conference/components/native/Conference.js

```diff
@@ -484,15 +473,7 @@ function _mapStateToProps(state) {
          * @private
          * @type {boolean}
          */
-        _toolboxVisible: visible,
-
-        /**
-         * The indicator which determines whether the Toolbox is always visible.
-         *
-         * @private
-         * @type {boolean}
-         */
-        _toolboxAlwaysVisible: alwaysVisible
```

Comment:
can you explain why `alwaysVisible` needs to be removed ?

Reply:
I just wanted to stay consistent with the statuses. It felt weird for me that a component decides if we should show the toolbar or not, easily disregarding the alwaysVisible setting. Now not in this case, but i could just have falsly add a button, not considering this variable and just simply invoke the action with false, and it will hide the toolbox even though the alwaysVisible is true. I thought it's better if the reducer decides it.

After all I ended up not using that action, but create a new one (to toggle), but that uses the same logic, so I decided to leave it like that, maintaining consistency. But I'm happy to change it back if you insist